test/app-tap/datetime.test.lua

```diff
@@ -438,6 +441,25 @@ test:test("Simple tests for parser", function(test)
     test:ok(date.parse('1998', { format = '%Y', tzoffset = '+03:00' }) ==
             date.new({ year = 1998, tzoffset = 180 }))
 
+    -- Testcases with override timezone by setting tz.
+    -- Timezone is not specified in a parsed string.
+    test:ok(date.parse("1970-01-01T01:00:00", {tz = 'MSK'}) ==
+            date.new{year=1970, mon=1, day=1, hour=1, min=0, sec=0, tzoffset=180})
+    -- Timezone is specified in a parsed string as a military timezone.
+    test:ok(date.parse("1970-01-01T01:00:00Z", {tz = 'Europe/Moscow'}) ==
```

Comment:
Why do you use either double or single quotes for strings?
I don't get the pattern.
Here and below.

Reply:
Fixed (replaced single quotes to double quotes.).